activerecord/test/cases/associations/belongs_to_associations_test.rb

```diff
@@ -700,6 +700,17 @@ def test_custom_counter_cache
     assert_equal 17, reply.replies.size
   end
 
+  def test_replace_counter_cache
+    topic = Topic.create :title => "Zoom-zoom-zoom"
+    reply = Reply.create(:title => "re: zoom", :content => "speedy quick!")
```

Comment:
While we don't accept PR's to change hash syntax only, new tests should use the new hash syntax `Topic.create(title: 'Zoom-zoom-zoom)`